packages/wonder-blocks-dropdown/src/components/dropdown-core.tsx

```diff
@@ -624,7 +624,7 @@ class DropdownCore extends React.Component<Props, State> {
 
     handleKeyDown: (event: React.KeyboardEvent) => void = (event) => {
         const {enableTypeAhead, onOpenChanged, open, searchText} = this.props;
-        const keyCode = event.which || event.keyCode;
+        const key = event.key;
```

Comment:
Was running into an issue where the unit tests wouldn't have the expected behaviour after keypresses. Turns out `which` and `keyCode` are deprecated and user-events doesn't support it anymore (in tests, keyCode and which would return 0). (see pr description for more details!)

Reply:
Thanks for fixing this!